indico/modules/events/management/templates/settings.html

```diff
@@ -5,7 +5,8 @@
 
 {% block content %}
     {{ template_hook('event-actions', event=event) }}
-    {%- if indico_config.ENABLE_ROOMBOOKING and not event.has_ended and event.room and not event.room_reservation_links -%}
+    {%- if indico_config.ENABLE_ROOMBOOKING and not event.has_ended and event.room and not event.room_reservation_links
+    and not has_overlap -%}
```

Comment:
that's a weird indentation level for wrapping this. how about moving this logic into the RH class and just check e.g. `if show_no_booking_warning` here?